pkg/k8s/apis/cilium.io/v2alpha1/types.go

```diff
@@ -51,14 +51,17 @@ type CiliumEgressNATPolicyList struct {
 	Items []CiliumEgressNATPolicy `json:"items"`
 }
 
+// +kubebuilder:validation:Pattern=`^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\/([0-9]|[1-2][0-9]|3[0-2])$`
+type IPv4CIDR string
```

Comment:
Nice. I would move this down below where it's used

Reply:
Not gonna fix since this one is minor.